plugins/database/cassandra/cassandra.go

```diff
@@ -89,13 +90,24 @@ func (c *Cassandra) CreateUser(ctx context.Context, statements dbplugin.Statemen
 		return "", "", err
 	}
 
-	creationCQL := statements.CreationStatements
-	if creationCQL == "" {
+	var creationCQL string
+	switch len(statements.CreationStatements) {
+	case 0:
 		creationCQL = defaultUserCreationCQL
+	case 1:
+		creationCQL = statements.CreationStatements[0]
+	default:
+		return "", "", fmt.Errorf("expected 0 or 1 creation statements, got %d", len(statements.CreationStatements))
```

Comment:
Why can't this use the list?

Reply:
We could, I just decided that I didn't know enough about cassandra, mongo, etc to make the proper determination.  Since Cassandra has a SQL like syntax, it probably does make sense to take the list.